app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -190,29 +191,29 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
     }
 
 
-    private void setBackButtonOnClick(){
+    private void setBackButtonOnClick() {
         ImageButton backButton = this.findViewById(R.id.routesPageBackButton);
         backButton.setOnClickListener(v -> exitSelectToFrom());
     }
 
-    private void exitSelectToFrom(){
-        Intent exitSelectToFrom= new Intent(RoutesActivity.this,
+    private void exitSelectToFrom() {
+        Intent exitSelectToFrom = new Intent(RoutesActivity.this,
                 MainActivity.class);
 
         SelectingToFromState.reset();
 
         startActivity(exitSelectToFrom);
     }
 
-    private void openSearchPage(){
-        Intent openSearch= new Intent(RoutesActivity.this, SearchActivity.class);
+    private void openSearchPage() {
+        Intent openSearch = new Intent(RoutesActivity.this, SearchActivity.class);
         startActivity(openSearch);
     }
 
     public void getShuttle(View view) {
-        List<Shuttle> shuttles = mViewModel.getShuttles();
+        List<Shuttle> shuttles = mViewModel.getAllShuttles();
         String shuttleDisplayText = mViewModel.getShuttleDisplayText(shuttles);
-        this.content.setText(shuttleDisplayText);
+        // this.content.setText(shuttleDisplayText);
```

Comment:
why is this commented out

Reply:
I will change it in a separate PR to be a list so we can display it with the routeOptions.. But good catch I'll remove it from here